pkg/backend/display/testdata/not-truncated/failed-stacks-dont-rewrite-operations.json.interactive-100x80-plain.stdout.txt

```diff
@@ -0,0 +1,17 @@
+View in Browser (Ctrl+O): link
```

Comment:
I know you did not introduce this style of testing but I find this "overfitting" - such tests make it hard to introduce unrelated changes. For example, adding `askCopilot` output will require changes in all of these files.

Reply:
I think we prefer it because otherwise it is very easy for us to produce incoherent text output, as testing each portion of the rendering in isolation doesn't give a complete picture of what the user sees.

The PR adding askCopilot required a flag, though, but it should probably be "ShowLinkToCopilot" and not "SuppressLinkToCopilot"?

Reply:
Re: flag name: I followed the pattern of Suppress* names (SuppressOutputs, SuppressPermalink, SuppressTimings, SuppressProgress). All of them are ON by default unless explicitly suppressed. We want that for Copilot links as well.

Reply:
I suspect not - as we only want to show Copilot when the backend is `api.pulumi.com`?

Reply:
> I think we prefer it because otherwise it is very easy for us to produce incoherent text output, as testing each portion of the rendering in isolation doesn't give a complete picture of what the user sees.

Yeah this style is very intentional. It is meant to make changes to the output clear and obvious. (though admittedly the tagged output can be hard to parse).